package/network/services/dnsmasq/files/dnsmasqsec.hotplug

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+TIMEVALIDFILE=/tmp/ntp.d/dnsmasqsec
+
+[ "$ACTION" = stratum ] || exit 0
```

Comment:
In that case maybe only support step, see warning https://git.busybox.net/busybox/tree/networking/ntpd.c#n939